docs/ingest_data.md

```diff
@@ -0,0 +1,12 @@
+# Ingest data samples to Substra
```

Comment:
Since we're already using "add" everywhere when talking about all assets (included data samples), we should stick with it. 

I'm not against using ingest to avoid repetition but document names, titles and introductions should use "add" for clarity.

Reply:
Yes, thanks.